python/hail/dataset.py

```diff
@@ -2464,22 +2540,22 @@ def mendel_errors(self, output, fam):
         """Find Mendel errors; count per variant, individual and nuclear
         family.
 
-        **Implementation Details**
+        **Example**
```

Comment:
Okay, I've changed all such sections to **Examples** no matter if it's one or more example.